src/libaktualizr/primary/sotauptaneclient.h

```diff
@@ -139,14 +157,14 @@ class SotaUptaneClient {
   Uptane::ImagesRepository images_repo;
   Uptane::Manifest uptane_manifest;
   std::shared_ptr<INvStorage> storage;
-  std::shared_ptr<PackageManagerInterface> package_manager_;
   std::shared_ptr<HttpInterface> http;
+  std::shared_ptr<PackageManagerInterface> package_manager;
   std::shared_ptr<Uptane::Fetcher> uptane_fetcher;
   std::unique_ptr<ReportQueue> report_queue;
   Json::Value last_network_info_reported;
   Uptane::EcuMap hw_ids;
   std::shared_ptr<event::Channel> events_channel;
-  boost::signals2::connection conn;
+  boost::signals2::scoped_connection conn;
```

Comment:
I think there was a reason I didn't originally use a `scoped_connection`... but I can't figure out why not. Seems like a good choice.